### ISSUE TITLE
Use output format for output-templates with get/create/delete commands

### ISSUE THREAD

**Issue by k-butz** [reubenmiller/go-c8y-cli#289]
I've seen CLI is supporting output formats in the "list" commands, e.g.:

```sh
-> c8y devices list --outputTemplate "{duration: response.duration}" -o json -c
{"duration":215}

-> c8y devices list --outputTemplate "{duration: response.duration}" -o csv
294
```

However, output format seems not being supported for other sub-commands.

1. For `get` the output template is only shown with --raw and (partially) in table:
```sh
-> c8y devices get --id 13162319608 --outputTemplate "{duration: response.duration}" -o csv
,,,,,,

-> c8y devices get --id 13162319608 --outputTemplate "{duration: response.duration}" -o json -c
{}

-> c8y devices get --id 13162319608 --outputTemplate "{duration: response.duration}" -o table
| id                   | name                 | type                 | contenttype          | length               | owner                | lastupdated          |
|----------------------|----------------------|----------------------|----------------------|----------------------|----------------------|----------------------|
|                      |                      |                      |                      |                      |                      |                      |

-> c8y devices get --id 13162319608 --outputTemplate "{duration: response.duration}" --raw
{
  "duration": 235
}
```

2. Same on `create`: 
```sh
-> c8y inventory create --type kbTest --outputTemplate "{duration: response.duration}" -f -o csv
,,,,,,

-> c8y inventory create --type kbTest --outputTemplate "{duration: response.duration}" -f -o json -c
{}

-> c8y inventory create --type kbTest --outputTemplate "{duration: response.duration}" -f -o table
| id                   | name                 | type                 | contenttype          | length               | owner                | lastupdated          |
|----------------------|----------------------|----------------------|----------------------|----------------------|----------------------|----------------------|
|                      |                      |                      |                      |                      |                      |                      |

-> c8y inventory create --type kbTest --outputTemplate "{duration: response.duration}" -f --raw
{
  "duration": 231
}
```

3. For `delete` I can't get the output template working independently from output format (also not with raw):
```sh
-> c8y inventory delete --id 68162675406 --outputTemplate "{duration: response.duration}" -f --raw
✓ Deleted /inventory/managedObjects/68162675406 => 204 No Content

-> c8y inventory delete --id 23162672954 --outputTemplate "{duration: response.duration}" -f -o json -c
✓ Deleted /inventory/managedObjects/23162672954 => 204 No Content

-> c8y inventory delete --id 82162660637 --outputTemplate "{duration: response.duration}" -f -o csv
✓ Deleted /inventory/managedObjects/82162660637 => 204 No Content
```

Is it known/intended that the output format is not supported for these subcommands?

**Comment by reubenmiller** [comment]
There are a few things at play here.

### Problem 1 & 2: For get the output template is only shown with --raw and (partially) in table:

Turning off the view will give the correct behaviour, e.g. `--view off`. This would apply both for gets and creates. The main reason is both return a single object rather than a list of objects.

**CSV output**

```sh
c8y inventory get --id 129203 --outputTemplate "{duration: response.duration}" --view off -o csv
```

```text
102
```

**Table output**

```sh
c8y inventory get --id 129203 --outputTemplate "{duration: response.duration}" --view off
```

```text
| duration   |
|------------|
| 132        |
```

### Problem 3: For delete I can't get the output template working independently from output format

This will be solved after https://github.com/reubenmiller/go-c8y-cli/issues/290 is resolved. As the output is not printed as currently output templates are ignored if the body's response is empty (which is the case when you delete something).

## Summary

To fix the situation, the following items will need to be addressed:

* Should views be disabled automatically when an output template is being applied? (which makes sense if you think about it)
* Apply output templates even when the response body is empty (related to https://github.com/reubenmiller/go-c8y-cli/issues/290)

**Comment by k-butz** [comment]
Just wanted to add...I've realized I could also just pass the output template to `c8y util show` for the output formatting:

```sh
-> c8y inventory create --outputTemplate "{duration: response.duration, status: response.statusCode}" -f | c8y util show -o csvheader
duration,status
249,201
```
This is quite convenient and provides me the output options I was searching for.

**Comment by reubenmiller** [comment]
Resolved in [2.37.0](https://github.com/reubenmiller/go-c8y-cli/releases/tag/v2.37.0)
